org.compiere.db.postgresql.provider/src/org/compiere/db/DB_PostgreSQL.java

```diff
@@ -891,40 +680,119 @@ public Connection getDriverConnection (String dbUrl, String dbUid, String dbPwd)
 		getDriver();
 		return DriverManager.getConnection (dbUrl, dbUid, dbPwd);
 	}	//	getDriverConnection
+	
+	private Properties getPoolProperties() {	
+		//check property file from home
+		File userPropertyFile = new File(getPoolPropertiesFile());
+		URL propertyFileURL = null;
+		
+		if (userPropertyFile.exists() && userPropertyFile.canRead())
+		{			
+			try {
+				propertyFileURL = userPropertyFile.toURI().toURL();
+			} catch (Exception e) {
+				e.printStackTrace();
+			}
+		}
+			
+		if (propertyFileURL == null)
+		{
+			propertyFileURL = Ini.isClient()
+					? PostgreSQLBundleActivator.bundleContext.getBundle().getEntry("META-INF/pool/client.default.properties")
+							: PostgreSQLBundleActivator.bundleContext.getBundle().getEntry("META-INF/pool/server.default.properties");						

```

Comment:
Please remove isClient check and always load from server.default.properties

Reply:
Done! I've also removed the actual client properties file.